packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -623,6 +630,45 @@ export class BiconomySmartAccountV2 extends BaseSmartContractAccount {
     return this.getPaymasterFeeQuotesOrData(userOp, buildUseropDto.paymasterServiceData);
   }
 
+  /**
+   *
+   * @description This function will return an array of supported tokens from the erc20 paymaster associated with the Smart Account
+   * @returns Promise<{@link SupportedToken}>
+   *
+   * @example
+   * import { createClient } from "viem"
+   * import { createSmartAccountClient } from "@biconomy/account"
+   * import { createWalletClient, http } from "viem";
+   * import { polygonMumbai } from "viem/chains";
+   *
+   * const signer = createWalletClient({
+   *   account,
+   *   chain: polygonMumbai,
+   *   transport: http(),
+   * });
+   *
+   * const smartAccount = await createSmartAccountClient({ signer, bundlerUrl, biconomyPaymasterApiKey }); // Retrieve bundler url from dasboard
+   * const tokens = await smartAccount.getSupportedTokens();
+   *
+   * // [
+   * //   {
+   * //     symbol: "USDC",
+   * //     tokenAddress: "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
+   * //     decimal: 6,
+   * //     logoUrl: "https://assets.coingecko.com/coins/images/279/large/usd-coin.png?1595353707",
+   * //     premiumPercentage: 0.1,
+   * //   }
+   * // ]
+   *
+   */
+  public async getSupportedTokens(): Promise<SupportedToken[]> {
+    const anyTransactionThatShouldSucceedForEveryone = await this.getEnableModuleData(DEFAULT_ECDSA_OWNERSHIP_MODULE);
```

Comment:
if the module is already enabled and you try to re-enable it then it should revert imo.

Reply:
this could just be sending 0 native tokens to self. data: 0x, value: 0, to: self

Reply:
Makes sense